release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2550,6 +2602,78 @@ and recreate it without a BFD profile. (link:https://bugzilla.redhat.com/show_bu
 
 // TODO: ^^^ Add BZ when created.
 
+[id="ocp-4-10-ran-ga-known-issues"]
+
+* The Kubelet service monitor scrape interval is currently set to a hard-coded value. This means that there is less available CPU resources for workloads.
```

Comment:
Has this been reviewed by SME? Personally I think this does not need to be in the release notes, as this bz is basically requesting a feature enhancement. If we are going with this bz in release notes, then I'm okay with the contents.

Reply:
/lgtm from Ken Young is here https://github.com/openshift/openshift-docs/pull/41683/files#r846178527

Reply:
Thanks. I'm okay in this case.